flags.go

```diff
@@ -234,6 +249,11 @@ func newApp() (app *cli.App) {
 					"This is effective when --disable-http2 is set.",
 			},
 
+			cli.IntFlag{
+				Name:  "max-idle-conns-per-host",
+				Value: 100,
```

Comment:
did you check pranjal results. At what value we got maximum performance"

Reply:
value is 100 only